primitives/file-key-verifier/src/types.rs

```diff
@@ -252,6 +253,7 @@ impl<const H_LENGTH: usize> AsRef<[u8; H_LENGTH]> for FileKey<H_LENGTH> {
 pub struct FileMetadata<const H_LENGTH: usize, const CHUNK_SIZE: u64, const SIZE_TO_CHALLENGES: u64>
 {
     pub owner: Vec<u8>,
+    pub bucket_id: Vec<u8>,
```

Comment:
Probably not convenient that `bucket_id` is the second argument here, but the third when instantiating `FileMetadata`